infra/nightly-resources/index.js

```diff
@@ -126,34 +132,50 @@ function groupByBenchmark(benchList) {
     return groupedBy;
 }
 
+// b may be undefined
+function compareTo(a, b) {
+    // if b is undefined, return a
+    if (b === undefined) {
+        return tryRound(a);
+    } else {
+        var difference = a - b;
+        var sign = difference < 0 ? "" : "+";
```

Comment:
why no `-` if it improves over main?

Reply:
added comment- - is there for negative number printing already